content/docs/get-started/install/migrating-2.0.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Migrating to Pulumi 2.0
+meta_desc: This page provides instructions for migrating to Pulumi 2.0
+no_on_this_page: true
+---
+
+## Migrating to 2.0
+
+Upgrading to 2.0 is simple. First, you will download the 2.0 CLI, which is required to use the 2.0 SDK. Then, update each of your Pulumi programs to utilize the new SDK. If you're using Javascript or Typescript, you'll also need to ensure you're using invokes asynchronously. We provide detailed instructions on each of these steps below.
+
+## Install Pulumi 2.0 CLI
+
+If you are upgrading to a beta release, obtain the latest binaries from the [versions]({{< relref "versions" >}}) page. Then, follow the [manual installation]({{< relref "./#manual-installation" >}}) steps for your operating system.
```

Comment:
Perhaps add a `> Note` here that once the final `2.0.0` release is available, users will also be able to install from all currently supported package managers (HomeBrew, Choclatey).